$NetBSD: patch-client_meson.build,v 1.1 2026/04/26 14:48:56 vins Exp $

Replace custom targets with local script to avoid missing
shared objects in generated executables. 

--- client/meson.build.orig	2025-06-23 14:06:37.000000000 +0000
+++ client/meson.build
@@ -114,3 +114,15 @@ cd_create_profile = executable(
   install : true,
   install_dir : bindir
 )
+
+run_cd_idt8 = [
+  join_paths(meson.source_root(), 'client', 'run.sh'),
+  join_paths(meson.build_root(), 'lib', 'colord'),
+  cd_idt8
+]
+
+run_cd_create_profile = [
+  join_paths(meson.source_root(), 'client', 'run.sh'),
+  join_paths(meson.build_root(), 'lib', 'colord'),
+  cd_create_profile
+]
